<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_DocumentSetup</class>
 <widget class="QWidget" name="Prefs_DocumentSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Document Setup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="sizeTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Language:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="languageComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="pageSizeTitleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Page Size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="pageSizeLabel">
           <property name="text">
            <string>Size:</string>
           </property>
           <property name="buddy">
            <cstring>pageSizeComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="pageSizeLayout">
           <item>
            <widget class="QComboBox" name="pageSizeComboBox"/>
           </item>
           <item>
            <widget class="QToolButton" name="pageSizeLinkToolButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="arrowType">
              <enum>Qt::NoArrow</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="pageOrientatioLabel">
           <property name="text">
            <string>Orientation:</string>
           </property>
           <property name="buddy">
            <cstring>pageOrientationComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="pageOrientationComboBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="pageUnitsLabel">
           <property name="text">
            <string>Units:</string>
           </property>
           <property name="buddy">
            <cstring>pageUnitsComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="pageUnitsComboBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="pageWidthLabel">
           <property name="text">
            <string>Width:</string>
           </property>
           <property name="buddy">
            <cstring>pageWidthSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="ScrSpinBox" name="pageWidthSpinBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="pageHeightLabel">
           <property name="text">
            <string>Height:</string>
           </property>
           <property name="buddy">
            <cstring>pageHeightSpinBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="ScrSpinBox" name="pageHeightSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="applySizesToAllPagesCheckBox">
         <property name="text">
          <string>Apply changes to all pages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="applySizesToAllMasterPagesCheckBox">
         <property name="text">
          <string>Apply changes to all master pages</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutTab">
      <attribute name="title">
       <string>Layout</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="pageLayoutTitleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Page Layout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="pageLayoutLayout">
         <item row="4" column="0">
          <widget class="QLabel" name="layoutFirstPageIsLabel">
           <property name="text">
            <string>First Page is:</string>
           </property>
           <property name="buddy">
            <cstring>layoutFirstPageIsComboBox</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="layoutFirstPageIsComboBox">
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="fourFoldRadioButton">
           <property name="text">
            <string>4-Fold</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="threeFoldRadioButton">
           <property name="text">
            <string>3-Fold</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="facingPagesRadioButton">
           <property name="text">
            <string>Facing Pages</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="singlePageRadioButton">
           <property name="text">
            <string>Single Page</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">pageLayoutButtonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>353</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="marginsTab">
      <attribute name="title">
       <string>Margins &amp;&amp; Bleeds</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>458</width>
            <height>540</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="marginsTitleLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Margins</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="NewMarginWidget" name="marginsWidget" native="true"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="applyMarginsToAllPagesCheckBox">
             <property name="text">
              <string>Apply changes to all pages</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="applyMarginsToAllMasterPagesCheckBox">
             <property name="text">
              <string>Apply changes to all master pages</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="bleedsTitleLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Bleeds</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="leftMargin">
              <number>5</number>
             </property>
             <property name="topMargin">
              <number>5</number>
             </property>
             <property name="rightMargin">
              <number>5</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="NewMarginWidget" name="bleedsWidget" native="true"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="saveTab">
      <attribute name="title">
       <string>Saving &amp;&amp; Undo</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="saveCompressedCheckBox">
         <property name="text">
          <string>Save in Compressed Format</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="emergencyCheckBox">
         <property name="text">
          <string>Try to save an &quot;Emergency File&quot; in case of a crash</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="autosaveTitleLable">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Autosave</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autosaveCheckBox">
         <property name="text">
          <string>Enable automatic saving of documents</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Interval:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="autosaveIntervalSpinBox">
             <property name="suffix">
              <string> min</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Number of files to keep:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="autosaveCountSpinBox">
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" colspan="3">
          <widget class="QRadioButton" name="autosaveDocRadio">
           <property name="text">
            <string>Save into the same directory as the document</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="autosaveDirRadio">
           <property name="text">
            <string>Save into:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="autosaveDirEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="changeAutoDir">
           <property name="text">
            <string>Change...</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="autosaveKeepCheckBox">
         <property name="text">
          <string>Keep backup files after closing the document</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showAutosaveClockOnCanvasCheckBox">
         <property name="text">
          <string>Show autosave countdown clock on the canvas</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="undoRedoTitleLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Undo / Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="undoCheckBox">
         <property name="text">
          <string>Enable Undo/Redo System</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="undoLengthLabel">
           <property name="text">
            <string>Action History Length:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="undoLengthSpinBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>NewMarginWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/newmarginwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>languageComboBox</tabstop>
  <tabstop>pageSizeComboBox</tabstop>
  <tabstop>pageSizeLinkToolButton</tabstop>
  <tabstop>pageOrientationComboBox</tabstop>
  <tabstop>pageUnitsComboBox</tabstop>
  <tabstop>pageWidthSpinBox</tabstop>
  <tabstop>pageHeightSpinBox</tabstop>
  <tabstop>applySizesToAllPagesCheckBox</tabstop>
  <tabstop>applySizesToAllMasterPagesCheckBox</tabstop>
  <tabstop>singlePageRadioButton</tabstop>
  <tabstop>facingPagesRadioButton</tabstop>
  <tabstop>threeFoldRadioButton</tabstop>
  <tabstop>fourFoldRadioButton</tabstop>
  <tabstop>layoutFirstPageIsComboBox</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>applyMarginsToAllPagesCheckBox</tabstop>
  <tabstop>applyMarginsToAllMasterPagesCheckBox</tabstop>
  <tabstop>saveCompressedCheckBox</tabstop>
  <tabstop>emergencyCheckBox</tabstop>
  <tabstop>autosaveCheckBox</tabstop>
  <tabstop>autosaveIntervalSpinBox</tabstop>
  <tabstop>autosaveCountSpinBox</tabstop>
  <tabstop>autosaveDocRadio</tabstop>
  <tabstop>autosaveDirRadio</tabstop>
  <tabstop>autosaveDirEdit</tabstop>
  <tabstop>changeAutoDir</tabstop>
  <tabstop>autosaveKeepCheckBox</tabstop>
  <tabstop>showAutosaveClockOnCanvasCheckBox</tabstop>
  <tabstop>undoCheckBox</tabstop>
  <tabstop>undoLengthSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autosaveDocRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>autosaveDirEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosaveDocRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>changeAutoDir</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosaveDirRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>autosaveDirEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autosaveDirRadio</sender>
   <signal>clicked(bool)</signal>
   <receiver>changeAutoDir</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="pageLayoutButtonGroup"/>
 </buttongroups>
</ui>
